letsencrypt/cli.py

```diff
@@ -612,7 +630,7 @@ def run(args, config, plugins):  # pylint: disable=too-many-branches,too-many-lo
     else:
         display_ops.success_renewal(domains, action)
```

Comment:
When `--dry-run` is expanded to support `run`, leaving `action == "reinstall"` will have some consequences that may be incorrect here.  But I don't think it matters for this PR.